sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_polling.py

```diff
@@ -4,8 +4,10 @@
 # Licensed under the MIT License.
 # ------------------------------------
 
+from typing import Optional
 from azure.core.polling.base_polling import (
     LongRunningOperation,
+    PipelineResponseType,
```

Comment:
This can't be imported here because it's defined in a type checking block. You'll need to redefine it in this file.